### DISCUSSION TITLE
[Feature]: Show all remaining files to be uploaded

### DISCUSSION THREAD

**Discussion by JVT038** [immich-app/immich#1665] [Feature Request]
### Feature detail

It would be nice to view all files that are yet to be uploaded on my phone. Just like OneDrive shows that in the Photos section, I would like to have this feature as well. 
Perhaps when you click on the remaining files counter in the backup settings, it could open a new section, showing a gallery containing those files?

### Platform

Mobile App

**Comment by Aikufurr** [comment]
This would be very handy, my backups get stuck with 9 remaining out of 7k and with the logs not outputting anything related to it I'm at a loss to find out which is causing it
